tests/test_bundle_run.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import subprocess
+import sys
+import tempfile
+import unittest
+
+import nibabel as nib
+import numpy as np
+import yaml
+from parameterized import parameterized
+
+from monai.transforms import LoadImage
+
+TEST_CASE_1 = [
```

Comment:
Could you make this test case a JSON file in https://github.com/Project-MONAI/MONAI/tree/dev/tests/testing_data, perhaps adds a yaml version if needed...

Reply:
Added both JSON and YAML test files.

Thanks.